bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -74,26 +74,27 @@ impl Client {
 
     /// The homeserver this client is configured to use.
     pub fn homeserver(&self) -> String {
-        RUNTIME.block_on(async move { self.client.homeserver().await.to_string() })
+        RUNTIME.block_on(async move { self.async_homeserver().await })
+    }
+
+    pub async fn async_homeserver(&self) -> String {
```

Comment:
Not sure if there is a convention for naming async vs non async methods in Rust. Maybe this non-bridged version should be `homeserver_string` to describe the conversion.